src/server.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Server start point
+ */
+require('require-self-ref')
```

Comment:
As long as it doesn't cause much trouble in the future, it's fine with me. Seems like it could be a brittle library, but pretty convenient.